### ISSUE TITLE
Need a way to get reason of error "No LU or Luis content parsed!"

### ISSUE THREAD

**Issue by dawwa** [microsoft/botframework-cli#790]
## Versions
@microsoft/botframework-cli/4.9.0 win32-x64 node-v12.16.3
Windows Server 2019

## Describe the bug
We see bf luis:convert fails when running in AzureDevOps Build Pipeline, but does not repro when running the same command locally.

I did a brief check on the source code, looks like this, is there some workaround we can get the reason why the result is empty?

```ts
        result = await LuisBuilder.build(luFiles, flags.log, flags.culture)
        if (!hasContent(result)) {
          throw new CLIError('No LU or Luis content parsed!')
        }
```

## Current behavior
We only see "No LU or Luis content parsed!" as one line error in the build log, but have no idea why it happens as we could not repro it locally.

## Expected behavior
We need to understand what's the real failure behind the error message.

**Comment by vishwacsena** [comment]
@dawwa we will work on improving the error messages. To get yourself unblocked, can you check to see if the files exist and are accessible to the CLI in your pipeline? This usually occurs when we read an empty file or a file that does not contain any LU content.

**Comment by dawwa** [comment]
@vishwacsena , @feich-ms helped to produce a private bit which prints more details, that helped discover the issue.

**TL;DR:**
MSBuild Exec interfered the stdin parameter of bf CLI with unexpected token  `∩╗┐` 
The proposed fix is to ignore stdin when --in option is used.
@feich-ms will push a fix into nightly feed until an official fix rolls out.

**Detail Rootcause**
Basically, the stdin is interfered by MSBUILD Target Exec command, which is with System.Text.UTF8 encoding, and by default, UTF-8 encodings in .NET use a Unicode Byte Order Mark at the beginning of the stream, which caused the `∩╗┐` we see from the stdin that bf CLI received.

Similar issues of other products mentioned about this problem here
https://github.com/microsoft/msbuild/issues/397
https://github.com/TypeStrong/typedoc/issues/133

**Comment by feich-ms** [comment]
Created a PR here https://github.com/microsoft/botframework-cli/pull/796

**Comment by feich-ms** [comment]
Verfied in my private feed that this issue was resolved https://msazure.visualstudio.com/One/_build/results?buildId=31208400&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dcf93f35-c1ba-5dce-0078-8d628250da3a&l=1005

**Comment by feich-ms** [comment]
fixed by PR #796

**Comment by feich-ms** [comment]
@dawwa, the fix is in the latest nightly build here https://botbuilder.myget.org/feed/botframework-cli/package/npm/@microsoft/botframework-cli. Please refer to this version CLI tool by setting npm registry to https://botbuilder.myget.org/F/botframework-cli/npm/ before the fix goes into next official release.
